test/e2e/network/networking_perf.go

```diff
@@ -27,15 +33,21 @@ import (
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
 	e2eservice "k8s.io/kubernetes/test/e2e/framework/service"
 	imageutils "k8s.io/kubernetes/test/utils/image"
+
+	"context"
+	clientset "k8s.io/client-go/kubernetes"
 )
 
 const (
 	// empirically derived as a baseline for expectations from running this test using kube-up.sh.
 	gceBandwidthBitsEstimate = int64(30000000000)
 	// on 4 node clusters, we found this test passes very quickly, generally in less then 100 seconds.
 	smallClusterTimeout = 200 * time.Second
+	// set a baseline for iperf2 bandwidth
+	eightBillion = 8_000_000_000
```

Comment:
@eth0XFEED what should we set here?

Reply:
Maybe once defining the baseline we can call this const with the unit (like oneGigaperSecond = 1*1024*etc

Reply:
👍